code/gui.py

```diff
@@ -0,0 +1,240 @@
+import os
+import tkinter as tk
+from clusterer import do_clustering
+from subprocess import Popen, PIPE
+from tkinter import messagebox
+from tkinter.filedialog import askopenfilename, askdirectory
+from VectorModelBuilder import VectorModelBuilder
+
+VMB_ARGS = (
+    ("Dataset", ""), 
+    ("Count Method", "ngram"), 
+    ("Weighting", "ppmi"), 
+    ("Output Directory", "../vector_data/"), 
+    ("Output File Name", "None"), 
```

Comment:
It might be better to call this `"<default>`" or something like that. It's sort of unclear to the user what "None" means.